cfme/tests/networks/test_sdn_tag_tagvis.py

```diff
@@ -1,12 +1,17 @@
 import pytest
+
+from cfme.cloud.provider.azure import AzureProvider
+from cfme.cloud.provider.ec2 import EC2Provider
+from cfme.cloud.provider.gce import GCEProvider
 from cfme.cloud.provider.openstack import OpenStackProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
 from cfme.networks.views import (CloudNetworkView, SubnetView, NetworkRouterView, SecurityGroupView,
-                                 NetworkPortView, BalancerView, FloatingIpView)
+                                 NetworkPortView, BalancerView, FloatingIpView, )
 from cfme.utils.appliance.implementations.ui import navigate_to
-from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
-
 
 pytestmark = [
+    pytest.mark.provider([AzureProvider, EC2Provider, GCEProvider, OpenStackProvider],
```

Comment:
@t-fischer There is no point to test tagvis relationships for all providers OpenStackProvider is good enough as it covers all available relationships

Reply:
so i can delete this pr